tlog-cosignature.md

```diff
@@ -63,7 +63,7 @@ The signature MUST be a 72-byte `timestamped_signature` structure.
     }
 
 "timestamp" is the time at which the cosignature was generated, as seconds since
```

Comment:
according to who? you might consider a more secure form of timestamp here.

Reply:
According to the witness. You can actually use the witnesses to obtain a form of secure timestamp.